src/templates/base/2019/base_ebook.html

```diff
@@ -92,26 +109,160 @@
     prince-footnote-policy: keep-with-line;
   }
 
+{% else %}
+
+{% set unit = 'mm' %}
+{% set spine = 25 %}
+{% set pageWidth = 148 %}
+{% set pageHeight = 210 %}
+{% if request.args.get('unit') != None %}{% set unit = request.args.get('unit') %}{% endif %}
+{% if request.args.get('spine') != None %}{% set spine = request.args.get('spine')|float %}{% endif %}
+{% if request.args.get('pageWidth') != None %}{% set pageWidth = request.args.get('pageWidth')|float %}{% endif %}
+{% if request.args.get('pageHeight') != None %}{% set pageHeight = request.args.get('pageHeight')|float %}{% endif %}
+{% set totalWidth = pageWidth + spine + pageWidth %}
+{% set pageAndSpine = pageWidth + spine %}
+{% set halfSpine =  spine / 2 %}
+
+#front-cover {
+  width: {{ pageWidth }}{{ unit }};
+  left: {{ pageAndSpine }}{{ unit }};
+}
+
+#front-cover .ha-logo-wrapper {
+  max-width: {{ spine }}{{ unit }};
+  height: {{ halfSpine }}{{ unit }};
+}
+
+#spine {
+  left: {{ pageWidth }}{{ unit }};
+  width: {{ spine }}{{ unit }};
+}
+
+#spine .ha-logo-wrapper {
+  max-width: {{ spine }}{{ unit }};
+  height: {{ halfSpine }}{{ unit }};
+}
+
+#back-cover {
+  width: {{ pageWidth }}{{ unit }};
+}
+
+#back-cover .ha-logo-wrapper {
+  max-width: {{ spine }}{{ unit }};
+  height: {{ halfSpine }}{{ unit }};
+}
+
+@prince-pdf {
+  prince-pdf-display-doc-title: true;
+}
+
+@page {
+  margin: 0;
+  padding: 0;
+  font-size: 5pt;
+  size: {{ totalWidth }}{{ unit }} {{ pageHeight }}{{ unit }};
+  marks: crop cross;
+  bleed: 3mm;
+  prince-trim: 5mm;
+}
+
+@page :left {
+  background: #5c687d;
+  background: transparent linear-gradient(#5c687d 70%, transparent 30%);
+  background: transparent url(/static/images/intro-background-fit-cover.svg) bottom left / 100% 100% no-repeat;
+  /* background: transparent url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxOTIwIiBoZWlnaHQ9IjEyNTAiIHZpZXdCb3g9IjAgMCAxOTIwIDEyNTAiIHByZXNlcnZlQXNwZWN0UmF0aW89Im5vbmUiPg0KICA8ZGVmcz4NCiAgICA8c3R5bGU+DQogICAgICBzdmd7YmFja2dyb3VuZDojZmZmfQ0KICAgICAgLmNscy0xe2ZpbGw6I2E4Y2FiYTtmaWxsLXJ1bGU6ZXZlbm9kZDtvcGFjaXR5Oi4wNX0NCiAgICA8L3N0eWxlPg0KICA8L2RlZnM+DQogIDxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS0yMzEgODloMjA4OHY2NTFsLTIwODggMzYwVjg5eiIvPg0KICA8cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik0tNDcwIDIzOWgyMDg4djY1MWwtMjA4OCAzNjBWMjM5eiIvPg0KICA8cGF0aCBkPSJNLTE0NC0xOWgyMDg4djY1MUwtMTQ0IDk5MlYtMTl6IiBmaWxsPSIjMWEyYjQ5IiBvcGFjaXR5PSIuNzUiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPg0KICA8cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik0tNDcwLTgzaDIwODh2NjUxTC00NzAgOTI4Vi04M3oiLz4NCiAgPHBhdGggZD0iTS0yMzItNDhoMjA4OHY2NTFMLTIzMiA5NjNWLTQ4eiIgZmlsbD0iI2YyZjJmMiIgb3BhY2l0eT0iLjA1IiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz4NCjwvc3ZnPg0K); */
```

Comment:
Can this be removed?

Reply:
Yup. I had to change it slightly to get it to stretch across the whole cover which is why it's a different SVG but not need to base64 encode it for performance reasons. Removed.